### ISSUE TITLE
RSS feed for react.dev/blog

### ISSUE THREAD

**Issue by flaksp** [reactjs/react.dev#5700]
RSS is still a thing. The old one (reactjs.org) had an RSS feed and it was very convenient.

**Comment by Je12emy** [comment]
Bump! I would like to keep up with the latest news and hopefully newly published pages on the "Learn" page. I think the arch wiki does something similar

**Comment by talovski** [comment]
Or add a newsletter at least. Right now the only way to keep up to date is to use Twitter, which is not ideal.

**Comment by MitchTalmadge** [comment]
Any luck on an RSS feed? Any hacks or workarounds?

**Comment by canac** [comment]
I built a scraper that generates an RSS feed based on CSS selectors for the titles and links for cases like this. As a temporary workaround until an official RSS feed exists, here's a scraped feed for the React blog: https://rss-scraper.deno.dev/feed.xml?url=https%3A%2F%2Freact.dev%2Fblog&itemSelector=a&titleSelector=h2

**Comment by MitchTalmadge** [comment]
Nice work @canac! That app is really awesome!

**Comment by kuchta** [comment]
@MitchTalmadge RSS autodiscovery is under /blog path.

**Comment by ap0h** [comment]
We use RSS Slack app which push news to our Slack channel. So yeah, it's still a thing!

**Comment by DamienCassou** [comment]
> @MitchTalmadge RSS autodiscovery is for some time under /blog path.

What do you mean by that @kuchta?

**Comment by kuchta** [comment]
> Any luck on an RSS feed? Any hacks or workarounds?

@MitchTalmadge That it's linked on the page react.dev/blob, not on react.dev homepage if that was the reason you had no luck.

**Comment by DamienCassou** [comment]
@kuchta I can't find any RSS/atom feed in /blog. My RSS reader also doesn't find anything.

**Comment by rickhanlonii** [comment]
Sorry for the delay, new RSS Feed is available here: https://react.dev/rss.xml

**Comment by kuchta** [comment]
@rickhanlonii Thank you Ricky ❤️ Could you please also link it to home and /blog page so that readers could auto-discover it? 🙏🏻

**Comment by rickhanlonii** [comment]
Sorry can you be more specific? ELI5

**Comment by LasseRosenow** [comment]
I think he means this:

```html
<link rel="alternate" type="application/rss+xml" title="React RSS Feed" href="https://react.dev/rss.xml" />
```

If this tag is available on the header at `react.dev` and `react.dev/blog`
Then RSS Readers will automatically find the RSS feed otherwise hidden behind `react.dev/rss.xml`.
For example in my case I was only able to add this feed because I know the exact domain from this issue.

**Comment by rickhanlonii** [comment]
Ah ok, fixed in https://github.com/reactjs/react.dev/pull/6805

**Comment by LasseRosenow** [comment]
Thank you very much, I can confirm it works :)
